CONTRIBUTING.md

```diff
@@ -15,6 +15,54 @@ propose changes to this document in a pull request.
 * Perform a [cursory search](https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+repo%3Aprojectatomic%2Fatomicapp)
   to see if a similar issue has already been submitted
 
+## Developers
+
+First of all, fork the [repository](https://github.com/projectatomic/atomicapp) and clone your github repository:
+
+```shell
+$ git clone https://github.com/<your-username>/atomicapp
+```
+
+### Install this project for development
+
+Goto cloned repo and Simply run following commands
+
+```shell
+$ sudo pip install virtualenvwrapper
+$ cd atomicapp
+$ make develop
+```
+
+Goto the Nuleculized app directory and run
+
+```shell
+$ workon atomicapp
```

Comment:
`workon` is a `virtualenvwrapper`'s construct. See `virtualenvwrapper` [docs](https://virtualenvwrapper.readthedocs.org/en/latest/)